lib/pbench/server/api/resources/api_key.py

```diff
@@ -26,18 +29,68 @@ def __init__(self, config: PbenchServerConfig):
             ApiSchema(
                 ApiMethod.POST,
                 OperationCode.CREATE,
+                query_schema=Schema(
+                    Parameter("name", ParamType.STRING, required=True),
+                ),
```

Comment:
As a User, if I want to create a key, I don't need to be concerned with naming or describing it (particularly if it is my only key).  I'm comfortable with us accepting and displaying a blank description.  (However, if you're not, it should be easy enough to generate a default (e.g., "API_KEY_\<n>", where n is the `id`).)

I share your instinct that the key ID otherwise has no value to the user, and so it should not be displayed (particularly if we don't reuse ID values...because the values will become a discontiguous range, which will likely engender user questions).

The bigger question in my mind is whether the UI should display the key value.  Practice on this varies widely.  In some systems, the key value is available _only_ on creation (with support for downloading it and lots of caveats about how this is the only opportunity to access it); in some cases it is hidden by default and only displayed by clicking a button; while in others it is just part of the table.  I like the idea of supporting the User in exporting the value (e.g., offering to download it and/or having a button which copies it to the clipboard), but the value of the key is just gibberish so, as long as we have alternatives, I don't see much value in displaying it on the screen.